src/libmongoc/src/mongoc/mongoc-error.h

```diff
@@ -122,7 +123,9 @@ typedef enum {
 
    /* An error related to initializing client side encryption. */
    MONGOC_ERROR_CLIENT_INVALID_ENCRYPTION_STATE,
-   MONGOC_ERROR_CLIENT_INVALID_ENCRYPTION_ARG
+   MONGOC_ERROR_CLIENT_INVALID_ENCRYPTION_ARG,
+
+   MONGOC_ERROR_TIMEOUT_INVALID
```

Comment:
Can you add to the errors.rst tables with the new domain/code?

Reply:
Done,

Reply:
Done.